fop/fom_generic.c

```diff
@@ -98,7 +99,8 @@ M0_EXPORTED(m0_rpc_item_generic_reply_rc);
 
 static bool fom_is_update(const struct m0_fom *fom)
 {
-	return m0_rpc_item_is_update(m0_fop_to_rpc_item(fom->fo_fop));
+	return !m0_dtm0_is_a_volatile_dtm(fom->fo_service) &&
```

Comment:
What happens if fo_service is not dtm service?

Reply:
@sergey-shilov please put a comment that !m0_dtm0_is_a_volatile_dtm(fom->fo_service) will guarantee that the rest of conditional will work with `!DTM0` services

Reply:
Done.